2025-07-17 20:02:55 | bybit: Exception during fetch | Exception: RuntimeError('Bybit fetch failed: No USDT tickers returned, status code 200 but data empty')
Traceback: Traceback (most recent call last):
  File "C:\Users\n.soshenko\PycharmProjects\ARB_SpotFuture_REST\exchanges\bybit.py", line 23, in fetch_bybit_prices
    raise RuntimeError("No USDT tickers returned, status code 200 but data empty")
RuntimeError: No USDT tickers returned, status code 200 but data empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\n.soshenko\PycharmProjects\ARB_SpotFuture_REST\core\price_collector.py", line 52, in fetch_live_prices
    data = fetch_func()
  File "C:\Users\n.soshenko\PycharmProjects\ARB_SpotFuture_REST\exchanges\bybit.py", line 27, in fetch_bybit_prices
    raise RuntimeError(f"Bybit fetch failed: {e}")
RuntimeError: Bybit fetch failed: No USDT tickers returned, status code 200 but data empty

2025-07-17 20:03:23 | coinex: Exception during fetch | Exception: RuntimeError('Coinex fetch failed: No USDT tickers returned, status code 200 but data empty')
Traceback: Traceback (most recent call last):
  File "C:\Users\n.soshenko\PycharmProjects\ARB_SpotFuture_REST\exchanges\coinex.py", line 23, in fetch_coinex_prices
    raise RuntimeError("No USDT tickers returned, status code 200 but data empty")
RuntimeError: No USDT tickers returned, status code 200 but data empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\n.soshenko\PycharmProjects\ARB_SpotFuture_REST\core\price_collector.py", line 52, in fetch_live_prices
    data = fetch_func()
  File "C:\Users\n.soshenko\PycharmProjects\ARB_SpotFuture_REST\exchanges\coinex.py", line 31, in fetch_coinex_prices
    raise RuntimeError(msg)
RuntimeError: Coinex fetch failed: No USDT tickers returned, status code 200 but data empty

